css-three-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+  justify-content: center;
+  align-items: center;
+}
+
+.row {
+  display: flex;
+  width: 90%;
+  justify-content: center;
```

Comment:
The only styling that should ever be applied to a `row` class is `display: flex`. If you need other behavior, you should make another class you can add in addition to `row` and apply your styling to that class.